### ISSUE TITLE
Cannot use import statement outside a module on v0.10.0

### ISSUE THREAD

**Issue by dbvcode** [feathersjs-ecosystem/feathers-reactive#193]
### Steps to reproduce

(First please check that this issue is not already solved as [described
here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#report-a-bug))

- [x] Tell us what broke. The more detailed the better.
- [ ] If you can, please create a simple example that reproduces the issue and link to a gist, jsbin, repo, etc.

### Expected behavior
Update feathers-reactive from v0.9.0 to v0.10.0 and I expected everything to keep working.
v0.9.0 is working fine. I keep using it for now.

### Actual behavior
I get the error:
```
/home/dbv/tg/node_modules/feathers-reactive/src/index.js:1
import Debug from 'debug';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.Module._load (/home/dbv/tg/packages/node/src/executors/node/node-with-require-overrides.ts:14:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.feathers-reactive (/home/dbv/tg/dist/apps/playground/webpack:/tradeguardian/external commonjs "feathers-reactive":1:1)
    at __webpack_require__ (/home/dbv/tg/dist/apps/playground/webpack:/tradeguardian/webpack/bootstrap:19:1)
```

### System configuration

I use a the feathers api and client in a Nrwl NX workspace and share the client in an angular web app and a nodejs client. Don't know if this is relevant here.

**Module versions** (especially the part that's not working):
    "@feathers-plus/cache": "^1.4.0",
    "@feathersjs/authentication": "^4.5.15",
    "@feathersjs/authentication-client": "^4.5.15",
    "@feathersjs/authentication-local": "^4.5.15",
    "@feathersjs/authentication-oauth": "^4.5.15",
    "@feathersjs/client": "^4.5.15",
    "@feathersjs/configuration": "^4.5.15",
    "@feathersjs/errors": "^4.5.15",
    "@feathersjs/express": "^4.5.15",
    "@feathersjs/feathers": "^4.5.15",
    "@feathersjs/socketio": "^4.5.15",
    "@feathersjs/socketio-client": "^4.5.15",
    "@feathersjs/transport-commons": "^4.5.15",
    "feathers-hooks-common": "^6.1.3",
    "feathers-memory": "^4.1.0",
    "feathers-mongodb": "^6.4.1",
    "feathers-mongoose": "^8.5.1",
    "feathers-profiler": "^0.1.5",
    "feathers-reactive": "^0.10.0",
    "feathers-shallow-populate": "^2.5.1",

**NodeJS version**:
v14.18.2

**Operating System**:
Linux

**Browser Version**:
-

**React Native Version**:
-

**Module Loader**:

**Comment by luketych** [comment]
Same issue

**Comment by fusillijerry89** [comment]
Please help with this issue

**Comment by bitflower** [comment]
I'm trying to get `0.10.0` to work right now too. What I've seen is that its main property in `package.json` points to `src` rather then `lib`. This makes my bundler go crazy because many functions are imported within files of the lib that are not exported. I wonder how the libs bundler itself can handle it. Maybe it doesn't and that's why we have the `src` as the main package entry?

**Comment by daffl** [comment]
Should be fixed now in v0.11.0 via https://github.com/feathersjs-ecosystem/feathers-reactive/pull/211
